descriptors/ModuleDescriptor-template.json

```diff
@@ -83,7 +83,7 @@
     },
     {
       "id": "circulation",
-      "version": "8.1",
+      "version": "8.2",
```

Comment:
The version in `circulation.raml` should be updated to this version as well.

Reply:
versions updated